utils/python/CIME/utils.py

```diff
@@ -542,7 +542,6 @@ def get_project(machobj=None):
 
 def setup_standard_logging_options(parser):
     helpfile = "%s.log"%sys.argv[0]
-    helpfile = os.path.join(os.getcwd(),os.path.basename(helpfile))
```

Comment:
For reasons that are utterly baffling to me, this os.getcwd() was causing an OSError (both on master and this branch).